desktop/cmp/input/JsonInput.scss

```diff
@@ -29,6 +29,25 @@
       box-sizing: content-box !important;
     }
   }
+
+  &__action-buttons {
+    position: absolute !important;
+    top: 0;
+    right: 6px;
+    z-index: 3;
```

Comment:
Suggest a comment on how we ended up at 3 here. 😄 

I see this works well when e.g. launched from a dialog. The rules for z-index always escape me - does this end up being relative in some sense (inside a "stacking context")?

Reply:
Simply put, that's there to place it "above" the code-mirror row and enable clicking on it. The code-mirror styles use `z-index: 2` internally for the rows of content - without this rule, clicking on the buttons selects the first row of content instead.

Just added a comment to the code to clarify.